inc/spbc-settings.php

```diff
@@ -3540,6 +3555,36 @@ function spbc_list_table__get_args_by_type($table_type)
                 ),
             );
             break;
+
+        case 'file_monitoring':
+            $args = array(
+                'id' => 'spbc_tbl__scanner_scan_file_monitoring',
+                'actions' => array (
+                    'current_snapshot' => array(
+                        'name' => 'Snapshots',
+                        'handler' => 'spbc_scanner_file_monitoring_show_current_snapshot(this);',
+                    ),
+//                    'view_bad' => array(
+//                        'name'    => 'View Suspicious Code',
+//                        'handler' => 'spbc_scanner_button_page_view_bad_event(this);',
+//                    ),
+//                    'approve'  => array('name' => 'Approve', 'handler' => 'spbc_scanner_button_page_approve(this);'),
+                ),
+                'func_data_prepare' => 'spbc_field_scanner__prepare_data__file_monitoring_files',
+                'func_data_total'   => 'spbc_scanner__file_monitoring_count',
+                'func_data_get'     => 'spbc_scanner_file_monitoring_data',
+                'if_empty_items' => __('No important files to monitor? Seriously?', 'security-malware-firewall'),
```

Comment:
Change this phrase to the neutral sound, for example just "No important files" or do no display this section without important files found.

Reply:
Done